addons/nightvision/config.cpp

```diff
@@ -13,8 +13,11 @@ class CfgPatches {
             "ACE_NVG_Gen2",
             /*"ACE_NVG_Gen3",*/
             "ACE_NVG_Gen4_Black",
+            "ACE_NVG_Gen4_Black_WP",
             "ACE_NVG_Gen4",
+            "ACE_NVG_Gen4_WP",
             "ACE_NVG_Gen4_Green",
+            "ACE_NVG_Gen4_Green_WP",
             "ACE_NVG_Wide_Black",
             "ACE_NVG_Wide",
             "ACE_NVG_Wide_Green"
```

Comment:
None of these classes are actually required to be here, want me to remove them?

Reply:
Nah they should be.